File: B_PL_Codes/10-Comando_if/inJava/if02.java

/*
Lógica de programação
Prof. Jonatha Costa
Exercício resolvido em java:
Escreva um programa que leia duas notas, calcule a média e verifique se o aluno está aprovado ou não,
considerando como critério de aprovação média 7,0;
*/
import java.util.Scanner;

public class if02 {
    public static void main(String[] args) {
        float nota1;
        float nota2;
        // Leitura de daos
        Scanner entrada_dado = new Scanner(System.in);
        System.out.println("Informe a primeira nota:");
        nota1 = entrada_dado.nextFloat();
        System.out.println("Informe a segunda nota:");
        nota2 = entrada_dado.nextFloat();

        // Declarando a média
        float media = (nota1 + nota2) / 2;
        // Teste e resultados
        if (media >= 7.0) {
            System.out.println("\nCandidato aprovado!\n Média obtida: " + media);
        } else {
            System.out.println("\nCandidato não aprovado!\n Média obtida: " + media);
        }

        entrada_dado.close();
    }
}

File: B_PL_Codes/10-Comando_if/inPy/if02.py
"""
Lógica de programação
Prof. Jonatha Costa
Exercício resolvido em python
Escreva uma programa que leia uma nota e verifique se aprovado ou não,
Escreva um programa que leia duas notas, calcule a média e verifique se o aluno está aprovado ou não,
considerando como critério de aprovação média 7,0;
Versão 1: Código básico para passo-a-passo, sem blocos de código.

*****************************************************
Exercício : Reescreva o código utilizando outra lógica, estratégia ou comandos.
*****************************************************
"""

# Ler dados
nota1 = float(input("Informe uma nota 1:\n"))
nota2 = float(input("Informe uma nota 2:\n"))
# Testar
if  ((nota1+nota2)/2) >= 7.0:
    print("Candidato aprovado!\n")
else:
    print("Candidado não aprovado!\n")

